src/__tests__/u-chartStorage.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * @jest-environment jsdom
+ */
+
+const chartStorage = require('../lib/chartStorage')
+require('@testing-library/jest-dom')
+

```

Comment:
`localStorage` is part of the `window` object, which is part of the DOM. I think you need to define `initDomFromFiles()` for this test suite and call it for each test to isolate them from each other. Otherwise, I think the localStorage items could clash for tests running in parallel.